packages/tradershub/src/components/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -0,0 +1,75 @@
+.wallets-modal-step-wrapper {
```

Comment:
@thisyahlen-deriv can we replace this quill?

Reply:
I leave it up to @aizad-deriv once we have the base. It will take time if we do it from scratch now.